study.md

```diff
@@ -95,11 +104,11 @@ would use to satisfy these requirements.
 In your own words, explain the role of application seed data.
 
 ```md
-<!-- your response here -->
+Application seed data allows you to add inital data after a database is created.
 ```
 
 Should you use seeds to create data to experiment with during development?
 
 ```md
-<!-- your response here -->
+Yes you should.
 ```
```

Comment:
No. Seed data is *only* for data *required* by app in order for it to function. Example data does not meet this criteria. It's convenient, not necessary.